buildSrc/src/main/kotlin/NativeUtils.kt

```diff
@@ -1,12 +1,23 @@
 /*
- * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
  */
 
 import org.gradle.api.*
 
 fun Project.posixTargets(): List<String> = nixTargets() + windowsTargets()
 
-fun Project.nixTargets(): List<String> = darwinTargets() + linuxTargets()
+fun Project.nixTargets(): List<String> = darwinTargets() + linuxTargets() + androidNativeTargets()
+
+fun Project.supportsAndroidNative(): Boolean = project.targetIsEnabled("androidNative")
```

Comment:
Q: probably the same should be done for `watchOS` above? In future PR (can do after this is merged)

Reply:
Yes, it is a good idea, feel free to open a PR